db/schema.rb

```diff
@@ -23,4 +23,10 @@
     t.datetime "updated_at",      null: false
   end
 
+  create_table "users", force: :cascade do |t|
```

Comment:
So in my opinion, the table name `users` should be reserved for the `User` class. (https://www.pivotaltracker.com/story/show/113903681)